src/routes/+layout.svelte

```diff
@@ -31,12 +33,17 @@
 		height: 100dvh;
 		width: 100vw;
 		background-color: black;
+		/* padding: 0 var(--spacing-24); */
 	}
 
 	.wrapper {
-		max-height: 915px;
+		/* max-height: calc(100vh - var(--spacing-24)); */
```

Comment:
Why did you decide to do this? :) I am OK with this altough I quite liked having it look a bit more like a phone screen (this was Samsung Galaxy S20 Ultra)

Reply:
I think that limiting the height to a specific size puts us in a situation where it may not look right on different (taller) devices, and there's no reason for that. The .root is limiting the height to 100vh so that makes sure we're constrained to the device height.
But also probably because since I'm working on a desktop, I want to be able to have as much viewing area as possible. 😉

Reply:
Oh the scrolling works because I have a super long chat with myself going to test that 😅
And the max-height line is currently commented out but I think when I added it I had some padding in the .root so I was making up for it.